plugins/example_plugin/.gitignore

```diff
@@ -1,3 +1,8 @@
 .packages
 .flutter-plugins
 pubspec.lock
+.flutter-plugins-dependencies
+package_config.json
+flutter_export_environment.sh
+Flutter-Generated.xcconfig
+GeneratedPluginRegistrant.swift
```

Comment:
For example_plugin please put anything that deviates from the standard Flutter plugin template rules in a separate section starting with a comment explaining why there are extra ignore rules as compared to the template. Anything checked in to the example will spread to other repos, where we can't clean it up later, so it should be clearly explained.

I would also rather you not touch every plugin in the unrelated PR. If you want to do this in a separate PR that's fine, but this change is unrelated to what this PR is about.

Reply:
reverted